crates/amaru-ledger/src/rules/transaction/inputs.rs

```diff
@@ -96,3 +96,318 @@ where
bc1ea6a418af3b9b8fef18bb65147872bf5037d28dda7b7b = hex::decode("ab008482582047a890217e4577ec3e6d5db161a4aa524a5cce3302e389ccb22b5662146f52ab02825820b185c756bf483fa7223967947f93e7f6e4e351782591467d99979908b6a2f6b200825820b185c756bf483fa7223967947f93e7f6e4e351782591467d99979908b6a2f6b201825820b185c756bf483fa7223967947f93e7f6e4e351782591467d99979908b6a2f6b2020184a300581d704e3b75eb68829ca9e63b541f3645f78afe8d119932a7a2528171e74901821a002dc6c0a2581c436941ead56c61dbf9b92b5f566f7d5b9cac08f8c957f28f0bd60d4ba14c5041594d454e54544f4b454e192cb4581c5cf48c225e057f8da0787eae1e2bc62a1cf1d2ee34c7e2a9498b19b5a148416767537461746501028201d81858ead87b9fd8799fd8799f9f581c3f2728ec78ef8b0f356e91a5662ff3124add324a7b7f5aeed69362f4581c17942ff3849b623d24e31ec709c1c94c53b9240311820a9601ad4af0581cba4ab50bdecca85162f3b8114739bc5ba3aaa6490e2b1d15ad0f9c66581c25aa4132c7ce7d8f96ee977cd921cba7681891d114d088449d1d63b2581c5309fa786856c1262d095b89adf64fe8a5255ad19142c9c537359e41ff1917701a001b77401a001b774018c818641a000927c0d8799f0a140aff021905dcd8799f9f581c1a550d5f572584e1add125b5712f709ac3b9828ad86581a4759022baff01ffffffffa300581d704e3b75eb68829ca9e63b541f3645f78afe8d119932a7a2528171e74901821a001e8480a1581c5cf48c225e057f8da0787eae1e2bc62a1cf1d2ee34c7e2a9498b19b5a14a4f7261636c654665656401028201d8185827d8799fd87b9fa3001b0000000dfaa9b200011b00000191f5ced5ba021b00000191f5ea4cfaffffa300581d704e3b75eb68829ca9e63b541f3645f78afe8d119932a7a2528171e74901821a002dc6c0a2581c436941ead56c61dbf9b92b5f566f7d5b9cac08f8c957f28f0bd60d4ba14c5041594d454e54544f4b454e1a000a92a0581c5cf48c225e057f8da0787eae1e2bc62a1cf1d2ee34c7e2a9498b19b5a14652657761726401028201d81858d0d87c9fd8799f9fd8799f581c3f2728ec78ef8b0f356e91a5662ff3124add324a7b7f5aeed69362f41a00021118ffd8799f581c17942ff3849b623d24e31ec709c1c94c53b9240311820a9601ad4af01a000233c0ffd8799f581cba4ab50bdecca85162f3b8114739bc5ba3aaa6490e2b1d15ad0f9c661a00021833ffd8799f581c25aa4132c7ce7d8f96ee977cd921cba7681891d114d088449d1d63b21a000221a5ffd8799f581c5309fa786856c1262d095b89adf64fe8a5255ad19142c9c537359e411a0001ee15ffff1925dbffff82581d605309fa786856c1262d095b89adf64fe8a5255ad19142c9c537359e411a00a229b4021a00175e18031a043724da081a043724620b5820ee8f2ca9b9a3e8cdc86ab2421fe721753bfd9bb2bfbf483a8e0a35d4b7ab28e20d8182582047a890217e4577ec3e6d5db161a4aa524a5cce3302e389ccb22b5662146f52ab020e81581c5309fa786856c1262d095b89adf64fe8a5255ad19142c9c537359e411082581d605309fa786856c1262d095b89adf64fe8a5255ad19142c9c537359e411a00827b8d111a00370c3f128782582047a890217e4577ec3e6d5db161a4aa524a5cce3302e389ccb22b5662146f52ab00825820b6dcfdc52df91296d060b4fee05c05340c51e89b16ff7033a53a35d2cd7949b0008258205c6d1dba0f74e55143a751d3f92433bcfbaa63d20b909f5dde8246db9f38144c00825820f0b861a084a983537de82d769ddf50876ef1bbb7b504690f45bc65e65d4de8f200825820878296eb7662c6a6e600f48941b3eae1e582bc05aacd0d635fcc96448a5e4e2600825820efd8a48f69e5270cb73b438067e1b566d39758329a45e5d6d36755f2c732618400825820537de728655d2da5fc21e57953a8650b25db8fc84e7dc51d85ebf6b8ea16559601").expect("failed to decode hex");
```

Comment:
It'd be better to move those into test data files, and rely on some macro like `include_bytes!` or something making use of it to:

1. Ease the readability of the supporting test code. 
2. Make the test vectors easier to find and share with others. 

See for example:

- https://github.com/pragma-org/amaru/tree/main/crates/amaru-consensus/tests/data/headers
- https://github.com/pragma-org/amaru/blob/main/crates/amaru-consensus/src/lib.rs#L51-L64
- https://github.com/pragma-org/amaru/blob/main/crates/amaru-consensus/src/consensus/store.rs#L206-L214